tests/validators/test_sanity_checks.py

```diff
@@ -22,3 +28,45 @@ def test_validate_production_one_non_nan_value():
     assert res["2022-01-01 00:00:00+00:00"] == 1
     assert res["2022-01-01 01:00:00+00:00"] == 0
     assert res["2022-01-01 02:00:00+00:00"] == 1
+
+
+def test_validate_production_is_plausible():
+    events = load_fixture("production_with_unplausible_productions")
+    res = validate_production_is_plausible(events)
+    assert res["2022-01-01 00:00:00+00:00"] == 0
+    assert res["2022-01-01 01:00:00+00:00"] == 1
+    assert res["2022-01-01 02:00:00+00:00"] == 1
+
+
+def test_validate_reasonable_time_production():
+    events = load_fixture("production_with_unreasonable_datetimes")
+    res = validate_reasonable_time_production(events)
+    assert res["1982-01-01 00:00:00+00:00"] == 0
+    assert res["2022-01-01 00:00:00+00:00"] == 1
+    assert res["2082-01-01 00:00:00+00:00"] == 0
+
+
+def test_validate_reasonable_time_exchange():
+    events = load_fixture("exchange_with_unreasonable_datetimes")
+    res = validate_reasonable_time_exchange(events)
+    assert res["1982-01-01 00:00:00+00:00"] == 0
+    assert res["2022-01-01 00:00:00+00:00"] == 1
+    assert res["2082-01-01 00:00:00+00:00"] == 0
+
+
+def test_validate_exchange_netflow_is_plausible():
+    events = load_fixture("exchange_with_unplausible_netflows")
+    res = validate_exchange_netflow_is_plausible(events)
+    assert res["2022-01-01 00:00:00+00:00"] == 0
+    assert res["2022-01-01 01:00:00+00:00"] == 0
+    assert res["2022-01-01 02:00:00+00:00"] == 1
+
+
+def test_validate_exchange_netflow_doesnt_exceed_capacity():
```

Comment:
Why can't this work with a fixture as well? Is it because of the need to specify by hand the capacity?